docs/src/api/class-selectors.md

```diff
@@ -202,13 +202,27 @@ Script that evaluates to a selector engine instance. The script is evaluated in
 
 Script that evaluates to a selector engine instance. The script is evaluated in the page context.
 
+### option: Selectors.register.script
```

Comment:
```suggestion
### option: Selectors.register.path
```

I feel like this was broken before and should be like this.

Reply:
Updated.